app.py

```diff
@@ -1,25 +1,95 @@
-"""Final Project"""
+# pylint: disable=no-member
+"""Main app"""
 import os
 import flask
-from flask_cors import CORS
+from flask import render_template, redirect, flash, request
+from flask_login import login_required, current_user, login_user, logout_user
 from dotenv import find_dotenv, load_dotenv
-from models import db
+from models import db, User, loggingIn
 
+load_dotenv(find_dotenv())
 app = flask.Flask(__name__)
-CORS(app)
 
-load_dotenv(find_dotenv())
 app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URL")
+app.config["SECRET_KEY"] = os.getenv("SECRET_KEY")
 app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+app.config["SQLALCHEMY_POOL_SIZE"] = 100
+
+loggingIn.init_app(app)
+loggingIn.login_view = "login"
 
 db.init_app(app)
 with app.app_context():
     db.create_all()
 
-# routes go here
+# ROUTES
```

Comment:
It looks like your flask routes are returning templates, but instead they should be returning json data. Our back-end app is only for returning data that our react app will call using the fetch api.

They should be something like return flask.jsonify(data);